File: tests/rust/tests-rust-testing-selenium/tests/ui/register_form_tests.rs
use allure_cargotest::allure_test;
use tests_rust_testing_selenium as tests;

#[allure_test(name = "Register form fields and submit are visible")]
#[tokio::test]
async fn register_form_is_mounted() {
    tests::layer_ui(
        "Register form mount",
        "Authentication",
        "Register form",
        "normal",
    );
    allure_rust_commons::tag("mock");
    tests::with_browser(|| async {
        tests::RegisterPage::default()
            .open_page()
            .await
            .should_show_register_form()
            .await
            .should_have_form_title("Register")
            .await;
    })
    .await;
}

File: tests/csharp/tests-csharp-nunit-restsharp-selenium/Tests/Ui/RegisterFormTests.cs
using Allure.Net.Commons;
using Allure.NUnit.Attributes;
using Tests;

namespace Tests.Ui;

[AllureLabel("layer", "ui")]
[AllureEpic("Authentication")]
[AllureFeature("Register form")]
[AllureSeverity(SeverityLevel.normal)]
[AllureSuite("Register form mount")]
public sealed class RegisterFormTests : TestBase
{
    [Test]
    [Category("ui")]
    [Category("mock")]
    [AllureName("Register form fields and submit are visible")]
    public void RegisterFormIsMounted()
    {
        RegisterPage.OpenPage()
            .ShouldShowRegisterForm()
            .ShouldHaveFormTitle("Register");
    }
}
